src/constants/tokenLists/bsc.mainnet.tokenlist.ts

```diff
@@ -0,0 +1,47 @@
+export const BSC_MAINNET_TOKEN_LIST = {
```

Comment:
<img width="581" alt="Screenshot at Aug 06 4-34-30 PM" src="https://user-images.githubusercontent.com/4060207/128490565-de3a3f29-4552-42a9-838c-6c45e971bbb8.png">

We also need to update this modal

Reply:
agree